com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/json/references/messages.properties

```diff
@@ -9,7 +9,7 @@
 #    ModelSolv, Inc. - initial API and implementation and/or initial documentation
 ###############################################################################
 
-error_missing_reference= Invalid Reference - SwagEdit was unable to resolve the reference. \n\
+error_missing_reference= Invalid Reference - KaiZen Editor was unable to resolve the reference. \n\
```

Comment:
This message is actually used by Validator, but we should change phrasing to `Unable to resolve the reference`
See description:
> "Unable to resolve the reference" is fine. We don't need to say "KaiZen OpenAPI Editor is unable to resolve the reference."